File: src/Web/SWARM.PuP.Web/Code/ViewModels/LobbySearchResult.cs
using System.Collections.Generic;
using SWARM.PuP.Web.Models;

namespace SWARM.PuP.Web.ViewModels {
    public class LobbySearchResult {
        public int[] Counts { get; set; }
        public IEnumerable<Lobby> Result { get; set; }
    }
}

File: src/Android/api/src/main/java/com/swarmnyc/pup/viewmodels/LobbySearchResult.java
package com.swarmnyc.pup.viewmodels;

import com.swarmnyc.pup.models.Lobby;

import java.util.List;

public class LobbySearchResult {
    private int[]       counts;
    private List<Lobby> result;

    public int[] getCounts()
    {
        return counts;
    }

    public List<Lobby> getResult()
    {
        return result;
    }
}
